queries.md

```diff
@@ -4,76 +4,106 @@
 
 ### 1. All the companies whose name match 'Babelgum'. Retrieve only their `name` field.
 
+Filter: {name: "Babelgum"}
+Project: {name: 1, _id: 0}
 <!-- Your Code Goes Here -->
 
 ### 2. All the companies that have more than 5000 employees. Limit the search to 20 companies and sort them by **number of employees**.
+    Filter: {number_of_employees: {$gt: 5000} }
+    Sort: {number_of_employees: 1}
+    Limit: 20
 
 <!-- Your Code Goes Here -->
 
 ### 3. All the companies founded between 2000 and 2005, both years included. Retrieve only the `name` and `founded_year` fields.
 
+Filter: {founded_year: {$gte: 2000, $lte: 2005 }}
+Project: {name: 1, founded_year: 1, _id: 0}
 <!-- Your Code Goes Here -->
 
 ### 4. All the companies that had a Valuation Amount of more than 100.000.000 and have been founded before 2010. Retrieve only the `name` and `ipo` fields.
 
+Filter: {"ipo.valuation_amount": {$gte: 100000000}}
```

Comment:
`{$and: [{'ipo.valuation_amount': {$gte: 10000000}}, {founded_year:{$lt: 2010}}]}`